binding/java/src/ai/picovoice/rhino/Rhino.java

```diff
@@ -260,7 +273,7 @@ public Rhino build() throws RhinoException {
                 throw new RhinoException(new IllegalArgumentException("Sensitivity value should be within [0, 1]."));
```

Comment:
you should convert all these wrapped exceptions to the relevant RhinoException. IOException -> RhinoIOException, IllegalArgumentException -> RhinoInvalidArgumentException

Reply:
Also, in `GetInference` we catch a generic Exception and turn it into a RhinoException - this is no longer necessary. All native functions throw Rhino Exceptions by default now.